contrib/kind.sh

```diff
@@ -1047,9 +1047,6 @@ install_metallb() {
   fi
   git clone https://github.com/metallb/metallb.git
   pushd metallb
-  # temporary fix for metallb issue
```

Comment:
@andreaskaris since you are here :) The kind.sh installing metallb has 2 issues:

1. It always tries to git clone metallb, so if you run kind.sh with -mlb twice it fails because git clone complains directory already exists
2. kin.sh git clones metallb to the contrib directory assumingt kind.sh is executed from there, polluting the working dir (and my IDE tries to use its go.mod lol). We should clone it somewhere temporary and remove it when we are done?

Do you mind fixing these things as well?

Reply:
sure, will do that today

Reply:
done